src/main/webapp/app/instructor-course-dashboard/tutor-leaderboard/tutor-leaderboard.component.html

```diff
@@ -4,18 +4,16 @@
             <tr>
                 <th>ID</th>
                 <th>{{ 'arTeMiSApp.instructorCourseDashboard.tutorLeaderboard.name' | translate }}</th>
-                <th>{{ 'arTeMiSApp.instructorCourseDashboard.tutorLeaderboard.username' | translate }}</th>
```

Comment:
Please keep the username.

Reply:
I only wanted to drop the database ID, not the username

Reply:
Done